src/core/cluster.py

```diff
@@ -455,6 +468,13 @@ def controller_quorum_uris(self) -> str:
             )
         return ""
 
+    @property
+    def bootstrap_controller(self) -> str:
```

Comment:
I see that `controller_quorum_uris` are not being used anymore for the workload and they have been replaced by a combination of `bootstrap_controller` and `kraft_unit_id` keys.

If this is the expected use of them, there are a few things that would need to be checked and fixed:
- Remove `controller_quorum_uris` from the codebase
- On `ClusterState._broker_status()` add the correct checks:
```python
    @property
    def _broker_status(self) -> Status:  # noqa: C901
        """Checks for role=broker specific readiness."""
        . . .

        if self.kraft_mode:
            if not (self.peer_cluster.bootstrap_controller and self.peer_cluster.bootstrap_unit_id):
                return Status.NO_QUORUM_URIS
            if not self.cluster.cluster_uuid:
                return Status.NO_CLUSTER_UUID
        . . .
```
- check [this logic](https://github.com/canonical/kafka-operator/pull/281/files#diff-9c0857a619bc92ee0018d5a2f1d49535ce24148f346c2dbeb310766d6c3bbb3cR117-R118) on `controller` handler